<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player</class>
 <widget class="QMainWindow" name="player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../срань/logo_player.png</normaloff>../../срань/logo_player.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 52, 52);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QScrollArea" name="music_scroller">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>321</width>
         <height>511</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(165, 165, 165);
</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>317</width>
          <height>507</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QSlider" name="volume_slider">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>520</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="content_slider">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>450</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(85, 255, 0);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="info_button">
       <property name="geometry">
        <rect>
         <x>1010</x>
         <y>540</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 255, 58);
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>info.png</normaloff>info.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="image">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>431</width>
         <height>361</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="playlists">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(165, 165, 165);</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>380</y>
         <width>581</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="playing_song_title">
          <property name="font">
           <font>
            <family>Arial Rounded MT Bold</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="playing_song_artist">
          <property name="font">
           <font>
            <family>Arial Rounded MT Bold</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="add_music_button">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>60</y>
         <width>158</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 255, 58);
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>NEW SONG</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add_playlist_button">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>10</y>
         <width>158</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 255, 58);
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>NEW PLAYLIST</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+N</string>
       </property>
      </widget>
      <widget class="QPushButton" name="play_button">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>480</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 255, 58);
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="right_track_button">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>480</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 255, 58);
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="left_track_button">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>480</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 255, 58);
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="change_color_button">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>540</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 255, 58);
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>change_color_theme.png</normaloff>change_color_theme.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="random_track_button">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>480</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(23, 255, 58);
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>random_track.png</normaloff>random_track.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
